src/app/features/activity-list/components/transaction-list/bitcoin-transaction/bitcoin-transaction-icon.tsx

```diff
@@ -0,0 +1,56 @@
+import { FiArrowDown as IconArrowDown, FiArrowUp as IconArrowUp } from 'react-icons/fi';
+
+import { Box, BoxProps, Circle, ColorsStringLiteral, Flex, color } from '@stacks/ui';
+
+import { BitcoinTransaction } from '@shared/models/transactions/bitcoin-transaction.model';
+
+import { BtcIcon } from '@app/components/icons/btc-icon';
+import { useCurrentBtcNativeSegwitAccountAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+import { isBitcoinTxInbound } from './bitcoin-transaction.utils';
+
+interface TransactionIconProps extends BoxProps {
+  transaction: BitcoinTransaction;
+}
+type BtcStatusColorMap = Record<string, ColorsStringLiteral>;
+
+const statusFromTx = (tx: BitcoinTransaction): string => {
+  if (tx.status.confirmed) return 'success';
+  return 'pending';
+};
+
+const colorFromTx = (tx: BitcoinTransaction): ColorsStringLiteral => {
+  const colorMap: BtcStatusColorMap = {
+    pending: 'feedback-alert',
+    success: 'brand',
+  };
+
+  return colorMap[statusFromTx(tx)] ?? 'feedback-error';
+};
+
+function IconForTx(address: string, tx: BitcoinTransaction) {
+  if (isBitcoinTxInbound(address, tx)) return IconArrowDown;
+  return IconArrowUp;
+}
+export function BitcoinTransactionIcon({ transaction, ...rest }: TransactionIconProps) {
+  const btcAddress = useCurrentBtcNativeSegwitAccountAddressIndexZero();
```

Comment:
Could we pass this in as a prop. Maybe later we need to use this component with other data, which will be difficult because the assumption is made it always uses the current btc address.